src/style.css

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  /* line-height: 14.06px; */
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.nav {
+  margin: 0 50px 0 0;
+}
```

Comment:
Avoid using specific side margins for elements like `.nav`. It can lead to unexpected layouts on different screen sizes. Consider using padding within the parent element or CSS Grid/Flexbox properties for spacing.